docs/src/pages/utils/popper/popper.md

```diff
@@ -7,9 +7,11 @@ components: Popper
 
 <p class="description">A Popper can be used to display some content on top of another. It's an alternative to react-popper.</p>
 
-Things to know when using the `Popper` component:
+Some important features of the `Popper` component:
 
-- Poppers rely on the 3rd party library [Popper.js](https://github.com/FezVrasta/popper.js) for positioning.
+- 🕷 Popper relies on the 3rd party library ([Popper.js](https://github.com/FezVrasta/popper.js)) for perfect positioning.
+- 💄 It's an alternative API to react-popper. It aims for simplicity.
+- 📦 Weight less than [10 KB gzipped](https://github.com/mui-org/material-ui/blob/master/.size-limit.js).
```

Comment:
Maybe: '- 📦 It's less than [10 KB gzipped](https://github.com/mui-org/material-ui/blob/master/.size-limit.js).'